kitty/pass_keys.py

```diff
@@ -26,58 +20,61 @@ def encode_key_mapping(window, key_mapping):
+    if direction == "left" and (left_neighbors and right_neighbors):
+        boss.active_tab.resize_window("narrower", amount)
     # only has left neighbor
-    elif direction == 'left' and left_neighbors:
-        boss.active_tab.resize_window('wider', amount)
+    elif direction == "left" and left_neighbors:
+        boss.active_tab.resize_window("wider", amount)
     # only has right neighbor
-    elif direction == 'left' and right_neighbors:
-        boss.active_tab.resize_window('narrower', amount)
+    elif direction == "left" and right_neighbors:
+        boss.active_tab.resize_window("narrower", amount)
 
     # has a neighbor on both sides
-    elif direction == 'right' and (left_neighbors and right_neighbors):
-        boss.active_tab.resize_window('wider', amount)
+    elif direction == "right" and (left_neighbors and right_neighbors):
+        boss.active_tab.resize_window("wider", amount)
     # only has left neighbor
-    elif direction == 'right' and left_neighbors:
-        boss.active_tab.resize_window('narrower', amount)
+    elif direction == "right" and left_neighbors:
+        boss.active_tab.resize_window("narrower", amount)
     # only has right neighbor
-    elif direction == 'right' and right_neighbors:
-        boss.active_tab.resize_window('wider', amount)
+    elif direction == "right" and right_neighbors:
+        boss.active_tab.resize_window("wider", amount)
 
     # has a neighbor above and below
-    elif direction == 'up' and (top_neighbors and bottom_neighbors):
-        boss.active_tab.resize_window('shorter', amount)
+    elif direction == "up" and (top_neighbors and bottom_neighbors):
+        boss.active_tab.resize_window("shorter", amount)
     # only has top neighbor
-    elif direction == 'up' and top_neighbors:
-        boss.active_tab.resize_window('taller', amount)
+    elif direction == "up" and top_neighbors:
+        boss.active_tab.resize_window("taller", amount)
     # only has bottom neighbor
-    elif direction == 'up' and bottom_neighbors:
-        boss.active_tab.resize_window('shorter', amount)
+    elif direction == "up" and bottom_neighbors:
+        boss.active_tab.resize_window("shorter", amount)
 
     # has a neighbor above and below
-    elif direction == 'down' and (top_neighbors and bottom_neighbors):
-        boss.active_tab.resize_window('taller', amount)
+    elif direction == "down" and (top_neighbors and bottom_neighbors):
+        boss.active_tab.resize_window("taller", amount)
     # only has top neighbor
-    elif direction == 'down' and top_neighbors:
-        boss.active_tab.resize_window('shorter', amount)
+    elif direction == "down" and top_neighbors:
+        boss.active_tab.resize_window("shorter", amount)
     # only has bottom neighbor
-    elif direction == 'down' and bottom_neighbors:
-        boss.active_tab.resize_window('taller', amount)
+    elif direction == "down" and bottom_neighbors:
+        boss.active_tab.resize_window("taller", amount)
```

Comment:
looks like these lines came from different formatter settings. please revert for a clean diff.